### ISSUE TITLE
Timeout issues

### ISSUE THREAD

**Issue by CAReeser** [MatthewFlamm/pynws#64]
I use your code but I get timeout exceptions quite often. Is this unusual?

**Comment by MatthewFlamm** [comment]
Yes.  I think because the NWS servers are sometimes flaky.  You'll need to handle the timeouts in some way.

**Comment by lymanepp** [comment]
> Yes. I think because the NWS servers are sometimes flaky. You'll need to handle the timeouts in some way.

Maybe we should add a retry option?
